homeassistant/components/philips_js/media_player.py

```diff
@@ -81,70 +85,69 @@ def _inverted(data):
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
This needs to change to the coroutine function `async_setup_platform`.

Reply:
Indeed. surprised it worked.

Reply:
Fixed in #46428